### DISCUSSION TITLE
[@types/tabulator-tables] RowContextMenuSignature is wrong?

### DISCUSSION THREAD

**Discussion by comman-fukui** [DefinitelyTyped/DefinitelyTyped#70212] [Issues with a @types package]
This signature seems wrong.
``` typescript
// DefinitelyTyped/DefinitelyTyped/blob/master/types/tabulator-tables/index.d.ts, line 1000
export type RowContextMenuSignature =
    | Array<MenuObject<RowComponent> | MenuSeparator>
    | ((e: MouseEvent, component: RowComponent) => MenuObject<RowComponent> | false | any[]);
```

I believe the exact signature would be as follows.
``` typescript
// DefinitelyTyped/DefinitelyTyped/blob/master/types/tabulator-tables/index.d.ts, line 1000
export type RowContextMenuSignature =
    | Array<MenuObject<RowComponent> | MenuSeparator>
    | ((e: MouseEvent, component: RowComponent) => Array<MenuObject<RowComponent> | MenuSeparator> | false);
```

Since the function is designed to build the menu procedurally, the return value should be the same as a static menu array.
In fact, when I try to return a MenuObject<RowComponent> or any[] from the function, it does not work correctly.

[a comment by an automated account is not shown]

**Comment by mike-lischke** [comment]
Tricky to say what is right here, without a documentation link or other information that proves the claim. What makes me lean towards the change is the fact that the any array is gone.
